### ISSUE TITLE
Vender pesadelos no Sparkly dá mais sonhos do que comprar pela loja da Loritta.

### ISSUE THREAD

**Issue by herocrife** [LorittaBot/DreamLand#70]
**Qual é o problema?**

Vou tentar dar um resumo. Na loja da Loritta, 100k de sonhos valem 20 reais, só que no Sparkly, 15 reais é 500 pesadelos. Na economia do server do Sparkly, você pode vender pesadelos por um valor, normalmente acima de 3k de sonhos. Mas ai é que tá, quando se converte 3k, 3,5k ou 4k para a economia da Loritta é dividido pela metade, ou seja, vendendo cada pesadelo por 1,5k de sonhos (Loritta). Agora, se você comprar 15 reais de pesadelos e vender tudo e converter para a Loritta, você pode conseguir os seguintes resultados: 500k, 750k, 1kk (Loritta) / 1kk, 1,5kk, 2kk (Sparkly). Ou seja, você pode conseguir 5x ou mais sonhos na Loritta comprando pesadelos e vendendo pelo SparklyPower. Seria isso proposital?

**Como reproduzir o problema?**

1. Comprar pesadelos.
2. Vender pesadelos no Sparkly.
3. ???
4. Profit.

**Considerações Finais**

Se isso for proposital, eu acho que seria meio injusto tanto para a Loritta quanto para as pessoas que compram sonhos na loja da Loritta, afinal é mais barato conseguir sonhos pelo Sparkly comprando pesadelos do que comprando sonhos na loja da Loritta.

**Comment by DarkAngel5965** [comment]
**<p>acho que isso não seria um bug</p>**

**Comment by MrSannyY** [comment]
> **<p>acho que isso não seria um bug</p>**

Sim, também vender pesadelos por terceiros não é um meio oficial, você pode comprar sonhos por terceiros e ter um valor bem a baixo do que está no site (claro que é proibido), tem meios oficiais de converter pesadelos em sonhos

**Comment by MrPowerGamerBR** [comment]
Como o @DarkAngel5965 e o @MrSannyY disseram: Não é um bug e você gastou todo o seu tempo escrevendo isso mesmo que a gente já tinha deixado claro no servidor do SparklyPower que isso não é um bug.

Como também os sonhos que são vendidos na loja do SparklyPower via `/lojacash` tem preços parecidos, se não quase idênticos, ao da Loritta.

Se você fica vendendo para players então não é um bug.

**Comment by MrPowerGamerBR** [comment]
Se a pessoa quer gastar o tempo dela tentando vender pesadelos no SparklyPower, o problema é dela.

Esse conceito de ser um bug é tão estúpido que me deixa até irritado, você também vai atrás de vendedores de matéria prima falando que "você deveria parar de vender madeira, tem gente que compra essa sua madeira, faz uma cadeira e vende por um preço mais alto!"? Pois é quase o mesmo conceito que você disse.

Aliás, se você é tão espertinho assim, me diga como você faria para arrumar isso, se a sua solução é "Banir vendas de pesadelos" então você está errado e não iremos fazer isso.
